appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -488,8 +488,10 @@ private YaVersion() {
   // - PEDOMETER_COMPONENT_VERSION was incremented to 2
   // For YOUNG_ANDROID_VERSION 196:
   // - WEBVIEWER_COMPONENT_VERSION was incremented to 8
+  // For YOUNG_ANDROID_VERSION 197:
```

Comment:
Generally I would recommend not bumping YOUNG_ANDROID_VERSION so long as you mark the PR editable by maintainers. This allows us to push a commit to your branch when we're ready to merge since we don't necessarily guarantee we can merge them in a particular order. Since this seems the most likely candidate right now I'll let it slide.

Reply:
Fixed, reverted these changes.